pkg/statsd/statsd_defaults_and_params.go

```diff
@@ -32,6 +32,9 @@ var DefaultTags = gostatsd.Tags{}
 // DefaultInternalTags is the default list of additional tags on internal metrics
 var DefaultInternalTags = gostatsd.Tags{}
 
+// DefaultHostname will be set to the default hostname of the system
+var DefaultHostname = getHost()
```

Comment:
This becomes a global thing, which we don't like.  It should be called from `AddFlags`